automate/models.py

```diff
@@ -23,22 +26,48 @@ class Project(BaseModel):
     primary_repo_type = models.CharField(
         max_length=50, choices=RepoType.choices, default=RepoType.GITHUB
     )
+    primary_token = models.TextField(
+        help_text="Primary Access token to make PR changes"
+    )
     secondary_repo = models.CharField(
         max_length=100,
         help_text="Name of repository that new changes would be pushed to",
     )
+    secondary_username = models.CharField(
+        max_length=200, help_text="The git username for the secondary repository"
+    )
     secondary_repo_url = models.CharField(
         max_length=500,
         help_text="The url repository that new changes would be pushed to",
     )
     secondary_repo_type = models.CharField(
         max_length=50, choices=RepoType.choices, default=RepoType.GITHUB
     )
+    secondary_token = models.TextField(
+        help_text="Secondary Access token to make PR changes"
+    )
     slug = models.SlugField(max_length=250, unique=True, blank=True)
+    base = models.CharField(
+        max_length=200,
+        help_text="The branch you want to target with PR changes. i.e., main",
+    )
 
     def save(self, *args, **kwargs):
         self.slug = slugify(self.name)
         super().save(*args, **kwargs)
 
     def __str__(self):
         return f"{self.name} Project"
+
+
+class History(BaseModel):
+    """History Model to retain payload information."""
+
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    pr_id = models.IntegerField()
+    action = models.CharField(max_length=30, null=True)
+    body = models.TextField()
+    url = models.URLField()
+    author = models.CharField(max_length=200)
+    merged_at = models.DateTimeField(null=True)
+    closed_at = models.DateTimeField(null=True)
```

Comment:
This is total not important in this PR, could be important in subsequent PR but not this one